contact_manager.py

```diff
@@ -105,12 +107,13 @@ def add_contact_to_db(self, db, contact_name, contact_fname, contact_lname, cont
         #open up the file
         with open('contacts.txt', 'a+') as f:
             #write the data
-            f.write("\nCONTACT: " + contact_info['contact'])
-            f.write("\n    First Name: " + contact_info['First Name'])
-            f.write("\n    Last Name: " + contact_info['Last Name'])
-            f.write("\n    Number: " + contact_info['Number'])
-            f.write("\n    Address: " + contact_info['Address'])
+            #f.write("\nCONTACT: " + contact_info['contact'])
+            #f.write("\n    First Name: " + contact_info['First Name'])
+            #f.write("\n    Last Name: " + contact_info['Last Name'])
+            #f.write("\n    Number: " + contact_info['Number'])
+            #f.write("\n    Address: " + contact_info['Address'])
             #rebuild the db
+            f.write("\n"+self.format_contact_info(contact_info))
```

Comment:
If this newline is removed Then we get an error when we try to add contact to file